packages/create-nx-workspace/bin/create-nx-workspace.ts

```diff
@@ -1128,7 +1133,11 @@ async function createApp(
     restArgs.packageManager = packageManager;
   }
 
-  const args = unparse(restArgs).join(' ');
+  const args = unparse({
+    ...restArgs,
+    standaloneApi: restArgs['standaloneApi'] ? 'true' : 'false',
+    routing: restArgs['routing'] ? 'true' : 'false',
```

Comment:
How come these 2 flags need special treatment here? Should they actually be in `restArgs`?

Reply:
The `unparse` function treats booleans differently to strings. 

It will convert a falsey arg to `--no-routing`. If the type is instead a string, it keeps the value as is `--routing=false`.

It may have been a caching issue (since upgrading to node 18, I've been facing a lot of them) but I was getting unexpected behaviour when the `--routing=false` was converted to `--no-routing` and handled by the `new` generator. A routing setup was still being created.

Reply:
Oh.. let's look into the weird handling of `--no-routing` separately? Not giving them special treatment here should work... There are plenty of other `boolean` flags so if `--no-` isn't being handled correctly, then that would be a larger issue.

Reply:
Sure thing!